tests_onnxmltools/test_lightgbm.py

```diff
@@ -16,6 +16,11 @@
 from skl2onnx.common.data_types import (
     SequenceType, DictionaryType, Int64TensorType, StringTensorType
 )
+import onnxmltools  # noqa
```

Comment:
Why these imports?

Reply:
To investigate which import of onnxmltools is failing.

Reply:
I see, you are not looking to check this in?

Reply:
I'd like to understand why onnxmltools is failing.